### PR TITLE
aws,packet: make tags more uniform

### PR DESCRIPTION
**Pull request by alban**
On Packet, tags are a list of strings.
On AWS, tags are a key-value map.

There are some default tags:
- ManagedBy = Lokomotive
- CreatedBy = Unspecified

and some suggestions:
- ManagedBy = Lokomotive
- CreatedBy = DevOpsTeam

-----

Related:
- https://github.com/kinvolk/lokomotive-kubernetes/pull/170
- https://github.com/kinvolk/lokomotive-kubernetes/pull/176

### CODE REVIEW

File: docs/flatcar-linux/packet.md [hunk 1]
```diff
@@ -316,6 +316,7 @@ Lokomotive implements support for some [DNS providers](../dns/), if your provide
 | reservation_ids_default | Default hardware reservation ID for nodes not listed in the `reservation_ids` map. | "" | "next-available"|
 | certs_validity_period_hours | Validity of all the certificates in hours | 8760 | 17520 |
 | controller_clc_snippets [[1]](#clc-snippets-limitation) | Controller Container Linux Config snippets | [] | [example](../advanced/customization.md#usage) |
+| tags | Optional details to tag on Packet resources | `[]` | `["CreatedBy-DevOpsTeam"]` |
```
Comment (johananl) [inline docs/flatcar-linux/packet.md]: I suggest we use `:` as a separator between the "key" and the "value" in Packet tags. The hyphen is less clear semantically IMO. Additionally, hyphens are much more likely to exist as part of either the key or the value, which would make it hard to know where the key ends and the value begins.

File: docs/flatcar-linux/aws.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ Reference the DNS zone id with `"${aws_route53_zone.zone-for-clusters.zone_id}"`
 | service_cidr | CIDR IPv4 range to assign to Kubernetes services | "10.3.0.0/16" | "10.3.0.0/24" |
 | cluster_domain_suffix | FQDN suffix for Kubernetes services answered by coredns. | "cluster.local" | "k8s.example.com" |
 | certs_validity_period_hours | Validity of all the certificates in hours | 8760 | 17520 |
-| tags | Optional details to tag on AWS resources | `{}` | `{"CreatedBy" = "Devops team"}` |
+| tags | Optional details to tag on AWS resources | `{}` | `{"CreatedBy" = "DevOps team"}` |
```
Comment (invidian) [inline docs/flatcar-linux/aws.md]: That seem unrelated to this PR/commit.

**Review by invidian — COMMENTED** [review]
This seems like a duplicate of #170.

**Comment by invidian** [comment]
#170 has been merged, so this PR needs to be rebased.

**Review by invidian — APPROVED** [review]
Looks OK.

File: docs/flatcar-linux/packet.md (hunk 1 above)
Comment (alban) [reply docs/flatcar-linux/packet.md]: Updated.
